pkg/controllers/federatedcluster/util.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/kubewharf/kubeadmiral/pkg/util/resource"
 )
 
+var VirtualNodeLabels map[string]string
```

Comment:
Please don't use global variables to pass options directly. This is not unit testable.

Reply:
@wy-lucky you can refer to other controller-specific options such as `NSAutoPropExcludeRegexp` or `ClusterJoinTimeout`

Reply:
please take a look.